VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating Denial of Service attacks and syncing new nodes on the network.
+Ultimately, as the network becomes more heavily used, multigigabyte per day bandwidth is very realistic.
+
+### What does running a validator imply in terms of logistics? 
+
+A successful validator operation will require the efforts of multiple highly skilled individuals and continuous operational attention. This will be considerably more involved than running a bitcoin miner for instance.
+
+### How to handle key management?
+
+Validators should expect to run an HSM that supports ed25519 keys. Here are the current options:
+* YubiHSM 2
+* Ledger Nano S
+* Potential options
+* Ledger BOLOS SGX enclave
+* Thales nCypher support
+* Tendermint SGX enclave
+
+### What can validators expect in terms of operations? 
+
+Running effective operation is the key to avoiding unexpectedly unbonding or being slashed. This includes being able to respond to attacks, outages etc. Maintain security and isolation in your data center etc.
+
+### What are the maintenance requirements?
+
+Validators should expect to perform regular software updates to accommodate upgrades and bug fixes. There will inevitably be issues with the network early in its bootstrapping phase that will require substantial vigilance 
+
+### How can validators protect themselves from denial-of-service attacks?
+ 
+DDOS attacks occur when an attacker sends a flood of internet traffic to an IP address to prevent the server at the IP address from connecting to the internet.
+A DDOS attacker scan the network and learn the IP address of various validator nodes and try to disconnect them from communication with floods of traffic. 
+
+One recommended way to mitigate these risks is for validators to carefully structure their network topology in a so-called Sentry node architecture. 
+Validator nodes should only connect to full-nodes they trust because they operate them themselves or are run by other validators they no socially. A Validator node will typically run in a data center. Most data center provide direct links the networks of major cloud providers. The validator can use those links to connect to sentry nodes in the cloud. 
+
+Sentry nodes can be quickly spun up or change their ip addresses. Because the links to the sentry nodes are in private ip space, an internet based attacked cannot distrust them directly. This will ensure validator block proposals and votes always make to the rest of the network.
+
+It is expected that good operating procedures on that part of validators will complete mitigate these threats.
+
+### How can I join the testnet?
```

Comment:
Let's move this all the way to the beginning of the article.

Reply:
ok